tests/settings.py

```diff
@@ -2,7 +2,12 @@
 import os
 
 DEBUG = True
-TEMPLATE_DEBUG = DEBUG
+
+BASE_PATH = os.environ.get('BASE_PATH', os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..')))
+
+PROJECT_PATH = os.environ.get('PROJECT_PATH', os.path.join(
+    BASE_PATH))
```

Comment:
Is the os.path.join() necessary here? It looked useless to me, so I checked the documentation. Apparently this will append a trailing path separator? But is that important for any filesystems, (as opposed to URLS, say)?

PS. If you do change anything you can switch "have up updated" to "have been updated" in the commit message.